src/server/scripts/Northrend/Ulduar/Ulduar/boss_razorscale.cpp

```diff
@@ -25,72 +25,70 @@
+    SPELL_LIGHTINGBOLT_25                   = 64696,
+    SPELL_CHAINLIGHTNING_10                 = 64758,
+    SPELL_CHAINLIGHTNING_25                 = 64759,
+};
+
+#define SPELL_FLAMEBUFFET                   RAID_MODE(SPELL_FLAMEBUFFET_10, SPELL_FLAMEBUFFET_25)
+#define SPELL_FLAMEBREATH                   RAID_MODE(SPELL_FLAMEBREATH_10, SPELL_FLAMEBREATH_25)
+#define SPELL_BATTLE_SHOUT                  RAID_MODE(SPELL_BATTLE_SHOUT_10, SPELL_BATTLE_SHOUT_25)
+#define SPELL_LIGHTINGBOLT                  RAID_MODE(SPELL_LIGHTINGBOLT_10, SPELL_LIGHTINGBOLT_25)
+#define SPELL_CHAINLIGHTNING                RAID_MODE(SPELL_CHAINLIGHTNING_10, SPELL_CHAINLIGHTNING_25)
 #define REQ_CHAIN_COUNT                     RAID_MODE(2, 4)
 
-#define SPELL_DEVOURINGFLAME_SUMMON         63308
-//#define SPELL_DEVOURINGFLAME_GROUNDAURA_10  64709
-//#define SPELL_DEVOURINGFLAME_GROUNDAURA_25  64734
-//#define S_DEVOURINGFLAME_GROUNDAURA         RAID_MODE(SPELL_DEVOURINGFLAME_GROUNDAURA_10, SPELL_DEVOURINGFLAME_GROUNDAURA_25)
-//#define NPC_DEVOURINGFLAME                  34188
-#define SPELL_STORMSTRIKE                   51876
-#define SPELL_WHIRLWIND                     63808
-#define SPELL_LIGHTINGBOLT                  63809
-#define SPELL_CHAINLIGHTNING                64758
-
-#define NPC_DARK_RUNE_SENTINEL              33846
-#define NPC_DARK_RUNE_GUARDIAN              33388
-#define NPC_DARK_RUNE_WATCHER               33453
-#define NPC_EXPEDITION_ENGINEER             33287
-#define NPC_EXPEDITION_COMMANDER            33210
-//#define NPC_EXPEDITION_DEFENDER             33816
-//#define NPC_EXPEDITION_TRAPPER              33259
-#define NPC_RAZORSCALE                      33186
-//#define NPC_HARPOON_FIRE_STATE              33282
-
-#define GO_DRILL                            195305
-#define GO_HARPOON_GUN_1                    194519
-#define GO_HARPOON_GUN_2                    194541
-#define GO_HARPOON_GUN_3                    194542
-#define GO_HARPOON_GUN_4                    194543
-#define GO_BROKEN_HARPOON                   194565
-
-#define TEXT_GOSSIP_ACTION                  "We are ready to help!"
-#define TEXT_EE_AGGRO                       "Give us a moment to prepare to build the turrets."
-#define TEXT_EE_MOVE_OUT                    "Ready to move out, keep those dwarves off of our backs!"
-#define TEXT_EE_FIRES_OUT                   "Fires out! Let's rebuild those turrets!"
-
-#define TEXT_TURRET_READY                   "Harpoon Turret is ready for use!"
-#define TEXT_DEEP_BREATH                    "Razorscale takes a deep breath..."
-#define TEXT_GROUNDED_PERMANENTLY           "Razorscale grounded permanently!"
-
-#define CORDS_GROUND                        588.0f, -166.0f, 391.1f
-#define CORDS_AIR                           588.0f, -178.0f, 490.0f
-#define REPAIR_POINTS                       25
-
-enum eSay
+#define TEXT_TURRET_READY                   "Harpoon Turret is ready for use!" // Should be said by NPC_RAZORSCALE_CONTROLLER
```

Comment:
no

Reply:
Please read the comment on the line, it's old code, I haven't touched that (It's the only hardcode left) but needs additional things.